src/hotspot/share/classfile/stackMapFrame.hpp

```diff
@@ -136,6 +155,64 @@ class StackMapFrame : public ResourceObj {
   inline u2 max_stack() const                 { return _max_stack; }
   inline bool flag_this_uninit() const        { return _flags & FLAG_THIS_UNINIT; }
 
+  AssertUnsetFieldTable* assert_unset_fields() const {
+    return _assert_unset_fields;
+  }
+
+  void set_assert_unset_fields(AssertUnsetFieldTable* table) {
+    _assert_unset_fields = table;
+  }
+
+  bool satisfy_unset_field(Symbol* name, Symbol* signature) {
```

Comment:
Could you add a comment above this to say when it's called?  Is this called for putfield?